.github/actions/tscommon/src/file.ts

```diff
@@ -6,11 +7,21 @@ import process from "process";
 // We need to set the working directory to the tscommon/ directory
 // instead of the GITHUB_WORKSPACE.
 export function getGitHubWorkspace(): string {
-  const wdt = process.env["UNIT_TESTS_WD"] || "";
+  const wdt = process.env.UNIT_TESTS_WD || "";
   if (wdt) {
     return wdt;
   }
-  return process.env["GITHUB_WORKSPACE"] || "";
+  return process.env.GITHUB_WORKSPACE || "";
+}
+
+// safeFileSha256 returns the hex-formatted sha256 sum of the contents of an
+// untrusted file path.
+export function safeFileSha256(untrustedPath: string): string {
+  if (!safeExistsSync(untrustedPath)) {
```

Comment:
nit: you don't need this line. `safeReadFileSync()` will check that the file is safe to read. If it does not exists, the built-in read call https://github.com/slsa-framework/slsa-github-generator/blob/d10c919298e18c8afb2b5f68e1872ec98a750cfb/.github/actions/tscommon/src/file.ts#L82 will throw an exception.